akka-actor/src/main/scala/akka/actor/dungeon/ChildrenContainer.scala

```diff
@@ -140,7 +140,7 @@ private[akka] object ChildrenContainer {
     }
 
     override def toString =
-      if (c.size > 20) c.size + " children"
+      if (c.size > 20) c.size.toString + " children"
```

Comment:
Will this result in unnecessary boxing or wrapping (allocating objects) if it's a primitive `int`? In Java one would use `String.valueOf(i)`. I't probably not important here but might be something to watch out for in any hot path (and there we probably already had that problem unless Scala did something very clever with `int + string`.